### PR TITLE
Ports construction bags, makes sheet snatchers available, and gives chief engineer an atmos fan

### PR DESCRIPTION
**Pull request by BrumalLake**
## About The Pull Request
Ports [tgstation#44875](https://github.com/tgstation/tgstation/pull/44875)
Ports [tgstation#48410](https://github.com/tgstation/tgstation/pull/48410)
Adds construction bags that hold stock parts for machines and adds them to the rcd vendor and ce's locker
Adds a recipe for them and for sheet snatchers
Makes sheet snatchers hold 150 sheets
Fixes borg sheet snatchers only being able to hold 350 sheets when they can hold 500
Puts an atmos holofan in the ce's locker

## Why It's Good For The Game
Makes construction without an rcd and of non rcd structures more convenient

## Changelog
:cl:
add: construction bags from tg
add: sheet snatcher crafting recipe
add: atmos holofan in chief engineer's locker
balance: sheet snatcher holds 150 instead of 300 sheets
fix: borg sheet snatcher now holds true amount of sheets
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -272,6 +272,8 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
 	new/datum/stack_recipe("mining satchel", /obj/item/storage/bag/ore, 4), \
 	new/datum/stack_recipe("chemistry bag", /obj/item/storage/bag/chemistry, 4), \
 	new/datum/stack_recipe("bio bag", /obj/item/storage/bag/bio, 4), \
+	new/datum/stack_recipe("construction bag", /obj/item/storage/bag/bio, 4), \
```
Comment (Sarchutar) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]:
```suggestion
	new/datum/stack_recipe("construction bag", /obj/item/storage/bag/construction, 4), \
```

**Comment by eeSPee** [comment]
So the construction bag is just a inferior RPED?

Might be worth it if you let it hold floor tiles/iron rods/glass/cable/metal stacks.

**Comment by BrumalLake** [comment]
> 
> 
> So the construction bag is just a inferior RPED?
> 
> Might be worth it if you let it hold floor tiles/iron rods/glass/cable/metal stacks.

thats what the sheet snatcher is for, plus the construction bag is a tiny item
it also holds bs crystals circuits and cables as well

**Comment by KubeRoot** [comment]
> > So the construction bag is just a inferior RPED?
> > Might be worth it if you let it hold floor tiles/iron rods/glass/cable/metal stacks.
> 
> thats what the sheet snatcher is for, plus the construction bag is a tiny item
> it also holds bs crystals circuits and cables as well

The RPED, or at least BRPED, can hold bluespace crystals too. That said, construction bag can be very useful when building multiple machines. Would be particularly useful for science roundstart, to hold multiple R&D server circuit boards and circuit manipulators.

**Comment by jupyterkat** [comment]
i think the whole point of construction bags is that it can hold circuits, call it circuit bag instead lmao

**Comment by BrumalLake** [comment]
> 
> 
> i think the whole point of construction bags is that it can hold circuits, call it circuit bag instead lmao

its for constructing machines from scratch, not part replacement like rpeds or just holding circuits
im not gonna port something from tg then rename it

File: code/game/objects/items/storage/bags.dm [hunk 2]
```diff
@@ -245,7 +245,7 @@
 	icon = 'icons/obj/mining.dmi'
 	icon_state = "sheetsnatcher"
 
-	var/capacity = 300; //the number of sheets it can carry.
+	var/capacity = 150; //the number of sheets it can carry.
```
Comment (utilbird) [inline code/game/objects/items/storage/bags.dm]:
```suggestion
	var/capacity = 150 //the number of sheets it can carry.
```

**Comment by jupyterkat** [comment]
it's kinda useless for holding parts considering the brped exists

**Comment by BrumalLake** [comment]
> 
> 
> it's kinda useless for holding parts considering the brped exists

ok and? the normal rped is "kinda useless for holding parts considering the brped exists"

**Comment by jupyterkat** [comment]
that is also true

**Review by utilbird — APPROVED** [review]

**Pull request merged**